src/main/java/org/broadinstitute/consent/http/service/MatchService.java

```diff
@@ -178,36 +161,24 @@ protected List<Match> createMatchesForDataAccessRequest(DataAccessRequest dar) {
     public List<Match> createMatchesForConsent(String consentId) {
         List<Match> matches = new ArrayList<>();
         Consent consent = findConsent(consentId);
-        List<Dataset> dataSets = dataSetDAO.getDatasetsForConsent(consentId);
+        List<Dataset> dataSets = datasetDAO.getDatasetsForConsent(consentId);
         List<DataAccessRequest> dars = findRelatedDars(dataSets.stream().map(Dataset::getDataSetId).collect(Collectors.toList()));
         if (consent != null && !dars.isEmpty()) {
             Match match;
             for (DataAccessRequest dar : dars) {
                 try {
-                    match = singleEntitiesMatch(consent, dar);
+                    match = singleEntitiesMatchV1(consent, dar);
```

Comment:
out of curiosity - why are we doing V1 matching here and V2 elsewhere? is this a deprecated code path?

Reply:
In this PR, I am more focused in processing matches for Data Access Requests than for Consents (or datasets). This method is only called when we are creating a new consent from the `POST api/consent` endpoint. To minimize the impact of the overall scope of the PR, I left this in place. That endpoint is no longer used from the UI. It used to be used from ORSP as well, but I'm pretty sure we disabled that so this is essentially a dead code path and on the chopping block.